### PR TITLE
Webpack, Basic Routing, and Search Results Page

### PR DESCRIPTION
**Pull request by seangtkelley**

### CODE REVIEW

File: src/SearchPage.jsx [hunk 1]
```diff
@@ -102,7 +101,7 @@ class TrendingCardProt extends React.Component {
 
 let InfoModal = (props) => (
 
-  <div className="modal fade" id="howItWorksModal" tabindex="-1" role="dialog" aria-labelledby="exampleModalLongTitle" aria-hidden="true">
+  <div className="modal fade" id="howItWorksModal" tabIndex="-1" role="dialog" aria-labelledby="exampleModalLongTitle" aria-hidden="true">
```
Comment (aasnani) [inline src/SearchPage.jsx:104]: camelCase is quite satisfying

**Review by sjakati98 — COMMENTED** [review]
approved.

File: src/SearchPage.jsx (hunk 1 above)
Comment (sjakati98) [reply src/SearchPage.jsx:104]: snake_case_is_better

File: src/SearchPage.jsx [hunk 2]
```diff
@@ -1,3 +1,5 @@
+import React from 'react';
+
```
Comment (aasnani) [inline src/SearchPage.jsx:2]: cmon..man..

File: src/ContentPage.jsx [hunk 3]
```diff
@@ -1,11 +1,10 @@
+import React from 'react';
+
```
Comment (aasnani) [inline src/ContentPage.jsx:2]: plz rmv whitespace...mks code rn slow....tks

**Pull request merged**